src/main/kotlin/no/nav/syfo/sykmeldingstatus/redis/SykmeldingStatusRedisService.kt

```diff
@@ -0,0 +1,42 @@
+package no.nav.syfo.sykmeldingstatus.redis
+
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import redis.clients.jedis.Jedis
+import redis.clients.jedis.JedisPool
+
+class SykmeldingStatusRedisService(private val jedisPool: JedisPool) {
+
+    companion object {
+        private val log: Logger = LoggerFactory.getLogger(SykmeldingStatusRedisService::class.java)
+        private const val redisTimeoutSeconds: Int = 60
+    }
+
+    fun updateStatus(sykmeldingStatusEventDTO: SykmeldingStatusRedisModel, sykmeldingId: String) {
+        var jedis: Jedis? = null
+        try {
+            jedis = jedisPool.resource
+            jedis.setex(sykmeldingId, redisTimeoutSeconds, objectMapper.writeValueAsString(sykmeldingStatusEventDTO))
+        } catch (ex: Exception) {
+            log.error("Could not update redis for sykmeldingId {}", sykmeldingId, ex)
```

Comment:
Her sluker vi exception, men det er kanskje greit? For vi vil vel egentlig at ting skal funke selv om vi ikke får oppdatert redis? :)

Reply:
Tja, vi burde kanskje ha et sån tall på hvor ofte det feiler kanskje?